### PR TITLE
python3 migration: driver pi

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**

### CODE REVIEW

**Comment by philipwinklerdelmic** [comment]
This should be tested on the hardware (not a lot of simulation available).

File: src/odemis/driver/pi.py [hunk 1]
```diff
@@ -185,8 +186,8 @@ def _sendGetCommand(self, com, prefix="", suffix="\r\n\x03"):
         return (string): the report without prefix nor newline
         """
         assert(len(com) <= 10)
-        assert(len(prefix) <= 2)
```
Comment (pieleric) [inline src/odemis/driver/pi.py]: Why removing this check?

File: src/odemis/driver/pi.py [hunk 2]
```diff
@@ -743,6 +745,7 @@ def read(self):
         return self.file.read()
 
     def write(self, arg):
+        arg = arg.decode('latin1') if sys.version_info[0] >= 3 else arg
```
Comment (pieleric) [inline src/odemis/driver/pi.py]: If the file is opened in binanry mode, it should accept bytes (and return bytes), which would be more realistic, and would avoid this extra conversion. => `open(..., "bw+")`

**Comment by philipwinklerdelmic** [comment]
done

**Pull request merged**